extensions/adobe/experience/adcloud-experienceevent.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics-experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe AdCloud ExperienceEvent Base",
+  "type": "object",
+  "description":
+    "This AdCloud ExperienceEvent base schema includes the core/standard ExperienceEvent as well as the Adobe AdCloud ExperienceEvent Extensions.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/time-series",
+    "https://ns.adobe.com/xdm/context/identitymap",
+    "https://ns.adobe.com/xdm/context/experienceevent-advertising",
```

Comment:
I know we want our tooling to unroll all of the extensions that a leaf schema implements, but here in the repo for authoring purposes I don't think we should duplicate all of these.  It will be very difficult to maintain by hand!